infrastructure/todoapp-pod.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: todoapp
+  namespace: todoapp
+  labels:
+    app: todoapp
+spec:
+  containers:
+  - name: todoapp
+    image: nemereno/todoapp:3.0.0
+    ports:
+    - containerPort: 8080
+    livenessProbe:
+      httpGet:
+       path: api/healthz
+       port: 8080
+      initialDelaySeconds: 60
+      periodSeconds: 3
+    readinessProbe:
+      httpGet:
+       path: /ready
+       port: 8080
+      initialDelaySeconds: 5
+      periodSeconds: 3
```

Comment:
Incorrect path in livenessProbe and readinessProbe. You create endpoint `readiness/` and `liveness/`, but path in manifest  is `api/healthz` and `/ready`